context/context.go

```diff
@@ -521,10 +524,12 @@ func (ctx *FunctionContext) SetSyncRequest(w http.ResponseWriter, r *http.Reques
 }
 
 func (ctx *FunctionContext) SetEvent(inputName string, event interface{}) {
+	// klog.Errorf(inputName)
 	switch t := event.(type) {
 	case *common.BindingEvent:
 		be := event.(*common.BindingEvent)
 		ie := convertEvent(ctx, inputName, be.Data)
+
```

Comment:
This blank line should also be removed?
![image](https://user-images.githubusercontent.com/18525465/178253554-f593184b-26bb-4b65-884f-513e59bb46db.png)